subsys/logging/log_backend_uart.c

```diff
@@ -62,10 +62,24 @@ static void panic(struct log_backend const *const backend)
 {
 }
 
+static void dropped(const struct log_backend *const backend, u32_t cnt)
+{
+	int i;
+	struct device *dev = log_output.control_block->ctx;
+	static const char newline_str[] = "\r\n";
+
+	for (i = 0; i < cnt; i++) {
+		uart_poll_out(dev, '~');
+	}
+
+	char_out((u8_t *)newline_str, sizeof(newline_str), dev);
```

Comment:
sizeof(newline_str) evaluates to 3, is it intentional that you're also writing the terminating null byte?

Reply:
right, sending null wasn't intentional. Fixed to send sizeof - 1.